src/formats/internal/writer.c

```diff
@@ -7,6 +7,9 @@
 
 #include "formats/internal/writer.h"
 #include "utils/allocator.h"
+#ifdef N64_BUILD
+#include <libdragon.h>
+#endif
```

Comment:
This include is unused.
Please remove all `#include <libdragon.h>`'s from this PR, none should be needed.